embc-app/ClientApp/src/app/shared/components/side-box-info/side-box-info.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="card card-default sidebox-card mb-4">
+<div class="card card-default sidebox-card mb-4" style="border-left:0;border-right:0;border-top:0;">
```

Comment:
Did you consider using Bootstrap classes instead of inline styles?
https://getbootstrap.com/docs/4.0/utilities/borders/

Reply:
fixed below